applications/serial_lte_modem/prj.conf

```diff
@@ -75,21 +75,22 @@ CONFIG_SETTINGS_FCB=y
 CONFIG_FCB=y
 
 # nRF Cloud
-#CONFIG_NRF_CLOUD_IPV6=y
+CONFIG_NRF_CLOUD_IPV6=n
 CONFIG_NRF_CLOUD_MQTT=y
 CONFIG_NRF_CLOUD_FOTA=n
 CONFIG_NRF_CLOUD_AGPS=y
 CONFIG_NRF_CLOUD_AGPS_FILTERED=n
-CONFIG_NRF_CLOUD_PGPS=y
+CONFIG_NRF_CLOUD_PGPS=n
 CONFIG_NRF_CLOUD_LOCATION=y
 CONFIG_NRF_CLOUD_LOG_LEVEL_INF=y
 CONFIG_NRF_CLOUD_GPS_LOG_LEVEL_INF=y
+CONFIG_NRF_CLOUD_CLIENT_ID_SRC_INTERNAL_UUID=y
 CONFIG_DATE_TIME=y
 CONFIG_MODEM_INFO=y
 CONFIG_MODEM_INFO_ADD_DATE_TIME=n
-CONFIG_AT_MONITOR=y
+
+# UUID and JWT
```

Comment:
Please eloborate

Reply:
Comment for the settings is UUID and JWT, but the UUID setting has been moved to above group. The comment is no longer accurate. 

Rest of the message was a suggestion on how to go about that.

Reply:
JWT is required for #XUUID and nRF Cloud UUID get (as well as JWT sign).
So I moved this item out of nRF Cloud group in case of being set "n" when excluding nRF Cloud services.

Reply:
Somehow I thought that this was an existing comment and CONFIG_NRF_CLOUD_CLIENT_ID_SRC_INTERNAL_UUID was below that. My bad.